components/raftstore/src/store/fsm/peer.rs

```diff
@@ -633,6 +633,29 @@ where
                 PeerMsg::UpdateRegionForUnsafeRecover(region) => {
                     self.on_update_region_for_unsafe_recover(region)
                 }
+                PeerMsg::RaftLogFetched {
```

Comment:
Is it a typo ? `RaftEntryFetched`?

Reply:
cause there is a term`raftlog_gc`, so I change `entry` to `log`😂

Reply:
Oh, I forget to push the latest commit. What you have reviewed is stale, please check again